nntrainer/layers/conv2d_layer.cpp

```diff
@@ -281,8 +282,11 @@ void Conv2DLayer::finalize(InitLayerContext &context) {
   wt_idx[ConvParams::weight] =
     context.requestWeight(dim, weight_initializer, weight_regularizer,
                           weight_regularizer_constant, "filter", true);
-  wt_idx[ConvParams::bias] = context.requestWeight(
-    bias_dim, bias_initializer, WeightRegularizer::NONE, 1.0f, "bias", true);
+
+  if (disable_bias.empty() || disable_bias.get() == false) {
+    wt_idx[ConvParams::bias] = context.requestWeight(
```

Comment:
note) wt_idx[ConvParams::bias] will be default to be 0.

If someone tries to access this without requesting anything, this will falsely point to weights.

We will need to use default value of -1 or some big number to make it explicit error.

Reply:
I will make initialized to std::numeric_limits<unsigned int>::max() for all layers.

Reply:
Alright :) I suppose it's gotta be out of scope of this PR